src/components/Catalog.vue

```diff
@@ -250,13 +294,69 @@ export default {
     };
   },
   asyncComputed: {
+    collections: {
+      default: [],
+      lazy: true,
+      async get() {
+        const externalCollections = this.links.find(x => x.rel === "data");
+        if (externalCollections === undefined) {
+          return [];
+        }
+
+        try {
+          const rsp = await fetch(externalCollections.href);
```

Comment:
Can you use [fetchUri](https://github.com/radiantearth/stac-browser/blob/master/src/util.js#L6) here? This allows the proxy functionality to work, and provides an abstraction if there's any other modifications to fetching we want to do in the future.

Reply:
Oh, I just copy pasted from externalItems below: https://github.com/radiantearth/stac-browser/pull/59/files#diff-fbae1f7e04ad34c70ca90231db3dd296764917ad88929a0860f4fe50e8627320R364

So I guess we should also change it there?